arcgis-android-toolkit/src/main/java/com/esri/arcgisruntime/toolkit/control/JoystickSeekBar.kt

```diff
@@ -0,0 +1,231 @@
+/*
+ * Copyright 2019 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.esri.arcgisruntime.toolkit.control
```

Comment:
Maybe we should also keep this in package `com.esri.arcgisruntime.toolkit.ar`? It's most likely only ever be used in an AR context, right?

Reply:
Could do, but it feels like more of an all-purpose use control. Whatever you think is best, I'm happy to go with.

Reply:
Yes, I think it's actually ok to keep it in this package, as there is no mention of any AR in that code...